.travis.yml

```diff
@@ -6,7 +6,9 @@ cache:
 notifications:
   email: false
 node_js: '8'
-install: npm install
+install:
+  - sudo apt-get install libcairo2-dev libjpeg-dev libpango1.0-dev libgif-dev build-essential g++
```

Comment:
Why?

Reply:
These are the dependencies for canvas.

Reply:
See https://www.npmjs.com/package/canvas#installation

Reply:
Yes, but why do you need canvas in first place? Please explain

Reply:
See all previous builds. They fail because `canvas` (and it dependencies) are not installed.

https://travis-ci.org/stereobooster/react-ideal-image/jobs/389643221#L483

Reply:
Oh yes. It uses canvas to detect WebP support, I totally forgot that. On the other side we don't need full featured canvas here, we can mock it out. Mocking is easier than bring C++ dependencies to test one small feature